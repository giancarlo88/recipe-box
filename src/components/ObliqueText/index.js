import React from 'react'
import styled from 'styled-components'

const StyledObliqueText = styled.p`
font-style: oblique;
font-size: 12px;
padding: 7.5px 0;
`

const ObliqueText = ({text}) => (
  <StyledObliqueText>
    {text}
  </StyledObliqueText>
)

export default ObliqueText
