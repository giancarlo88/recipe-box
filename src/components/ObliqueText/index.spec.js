import React from 'react'
import { create } from 'react-test-renderer'
import { mount } from 'enzyme'

import ObliqueText from './'


describe('<ObliqueText />', () => {
  const text = 'Here is some oblique text'

  it('should match a snapshot', () => {
    const props = {
      text
    }
    const tree = create(<ObliqueText {...props} />)
    expect(tree).toMatchSnapshot()
  });


  it('should should render the text passed to it as a prop', () => {
    const component = mount(<ObliqueText text={text} />)
    expect(component.find('p').text()).toBe(text)
  });

});
