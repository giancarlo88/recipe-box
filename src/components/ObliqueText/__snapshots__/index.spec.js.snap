// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ObliqueText /> should match a snapshot 1`] = `
<p
  className="sc-bdVaJa hCJUJ"
>
  Here is some oblique text
</p>
`;
